source/core/render/trace.cpp

```diff
@@ -339,12 +339,13 @@ bool Trace::FindIntersection(ObjectPtr object, Intersection& isect, const Ray& r
         BBoxVector3d invdir;
         BBoxDirection variant;
 
-        Vector3d tmp(1.0 / ray.GetDirection()[X], 1.0 / ray.GetDirection()[Y], 1.0 /ray.GetDirection()[Z]);
+        Vector3d tmp(1.0 / ray.GetDirection()[X], 1.0 / ray.GetDirection()[Y], 1.0 / ray.GetDirection()[Z]);
         origin = BBoxVector3d(ray.Origin);
         invdir = BBoxVector3d(tmp);
         variant = (BBoxDirection)((int(invdir[X] < 0.0) << 2) | (int(invdir[Y] < 0.0) << 1) | int(invdir[Z] < 0.0));
 
-        if(object->Intersect_BBox(variant, origin, invdir, closest) == false)
+        // Note: At leaf when called. Suspect not strictly needed. However, without slightly slower in performance.
+        if((object->Intersect_BBox(variant, origin, invdir, closest) == false) && (sceneData->boundingMethod != 0))
```

Comment:
Due to the bounding method test being significantly less expensive than the bounding box intersection test, I would lean towards swapping around the two tests.